packages/quantic/force-app/main/default/lwc/quanticResultList/quanticResultList.js

```diff
@@ -1,19 +1,42 @@
-import {LightningElement, api, track} from 'lwc';
-import {registerComponentForInit, initializeWithHeadless} from 'c/quanticHeadlessLoader';
+import { LightningElement, api, track } from 'lwc';
```

Comment:
Please run the formatter on the imports.

Reply:
This is what my formatter does 😭, i have to un-run it.